students/thomas_sulgrove/lesson03/list_lab.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+##SP_Online_PY210 lesson03 list lab https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/list_lab.html
+
+
+def Series1():
+    #Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+    fruits = ["Apples", "Pears", "Oranges", "Peaches"] 
+    #Display the list (plain old print() is fine…)
+    print(fruits) 
+    #Ask the user for another fruit and add it to the end of the list
+    response = input("Please list another fruit: ")
+    fruits.append(response)
+    #Display the list.
+    print(fruits)
+    #Ask the user for a number and display the number back to the user and the fruit corresponding to that number (on a 1-is-first basis).
+    #Remember that Python uses zero-based indexing, so you will need to correct.
+    response = int(input("Please a number between 1 and " + str(len(fruits)) + ": "))
+    print(fruits[response - 1])
+    #Add another fruit to the beginning of the list using “+” and display the list
+    fruits = ["Alvocados"] + fruits #didnt know it was a fruit but it is. https://en.wikipedia.org/wiki/Avocado
+    print(fruits) 
+    #Add another fruit to the beginning of the list using insert() and display the list
+    fruits.insert(0,"Blueberries")
+    print(fruits)
+    #Display all the fruits that begin with “P”, using a for loop.
+    for produce in fruits:
+        if produce[0] == "P": #check first index of each interation
+            print(produce)
+    return fruits
+
+#Using the list created in series 1 above
+Fruit_List = Series1()
+
+def Series2(Fruit = Fruit_List.copy()): #make a copy of the list so you can make changes and not blow everything up.
```

Comment:
_never_ use mutable data for default value - it won't have an issue here but it can have really bad consequences